AsyncDisplayKit/ASTableView.mm

```diff
@@ -126,6 +126,11 @@ @interface ASTableView () <ASRangeControllerDelegate, ASDataControllerSource> {
   ASBatchContext *_batchContext;
 
   NSIndexPath *_pendingVisibleIndexPath;
+
+  NSIndexPath *_contentOffsetAdjustmentTop;
```

Comment:
This name is slightly unclear but tough to make more clear without being super long.  Could you add a small comment that explains its purpose?  E.g. "Set to the topmost fully-visible index path before executing editing commands, if automaticallyAdjustsContentOffset is enabled. "